src/geometry/euclidean/oriented_angle.lean

```diff
@@ -1765,3 +1772,190 @@ lemma oangle_sign_smul_add_smul_smul_add_smul (x y : V) (r₁ r₂ r₃ r₄ : 
+unoriented angle. -/
+lemma cos_oangle_eq_cos_angle {p p₁ p₂ : P} (hp₁ : p₁ ≠ p) (hp₂ : p₂ ≠ p) :
+  real.angle.cos (∡ p₁ p p₂) = real.cos (∠ p₁ p p₂) :=
+(o).cos_oangle_eq_cos_angle (vsub_ne_zero.2 hp₁) (vsub_ne_zero.2 hp₂)
+
+/-- The oriented angle at `p` between two points not equal to `p` is plus or minus the unoriented
+angle. -/
+lemma oangle_eq_angle_or_eq_neg_angle {p p₁ p₂ : P} (hp₁ : p₁ ≠ p) (hp₂ : p₂ ≠ p) :
+  ∡ p₁ p p₂ = ∠ p₁ p p₂ ∨ ∡ p₁ p p₂ = -∠ p₁ p p₂ :=
+(o).oangle_eq_angle_or_eq_neg_angle (vsub_ne_zero.2 hp₁) (vsub_ne_zero.2 hp₂)
+
+/-- The unoriented angle at `p` between two points not equal to `p` is the absolute value of the
+oriented angle. -/
+lemma angle_eq_abs_oangle_to_real {p p₁ p₂ : P} (hp₁ : p₁ ≠ p) (hp₂ : p₂ ≠ p) :
+  ∠ p₁ p p₂ = |(∡ p₁ p p₂).to_real| :=
+(o).angle_eq_abs_oangle_to_real (vsub_ne_zero.2 hp₁) (vsub_ne_zero.2 hp₂)
+
+/-- If the sign of the oriented angle at `p` between two points is zero, either one of the points
+equals `p` or the unoriented angle is 0 or π. -/
+lemma eq_zero_or_angle_eq_zero_or_pi_of_sign_oangle_eq_zero {p p₁ p₂ : P}
+  (h : (∡ p₁ p p₂).sign = 0) : p₁ = p ∨ p₂ = p ∨ ∠ p₁ p p₂ = 0 ∨ ∠ p₁ p p₂ = π :=
+begin
+  convert (o).eq_zero_or_angle_eq_zero_or_pi_of_sign_oangle_eq_zero h;
+    simp
+end
+
+/-- If two unoriented angles are equal, and the signs of the corresponding oriented angles are
+equal, then the oriented angles are equal (even in degenerate cases). -/
+lemma oangle_eq_of_angle_eq_of_sign_eq {p₁ p₂ p₃ p₄ p₅ p₆ : P} (h : ∠ p₁ p₂ p₃ = ∠ p₄ p₅ p₆)
+  (hs : (∡ p₁ p₂ p₃).sign = (∡ p₄ p₅ p₆).sign) : ∡ p₁ p₂ p₃ = ∡ p₄ p₅ p₆ :=
+(o).oangle_eq_of_angle_eq_of_sign_eq h hs
+
+/-- If the signs of two nondegenerate oriented angles between points are equal, the oriented
+angles are equal if and only if the unoriented angles are equal. -/
+lemma oangle_eq_iff_angle_eq_of_sign_eq {p₁ p₂ p₃ p₄ p₅ p₆ : P} (hp₁ : p₁ ≠ p₂) (hp₃ : p₃ ≠ p₂)
+  (hp₄ : p₄ ≠ p₅) (hp₆ : p₆ ≠ p₅) (hs : (∡ p₁ p₂ p₃).sign = (∡ p₄ p₅ p₆).sign) :
+  ∠ p₁ p₂ p₃ = ∠ p₄ p₅ p₆ ↔ ∡ p₁ p₂ p₃ = ∡ p₄ p₅ p₆ :=
+(o).oangle_eq_iff_angle_eq_of_sign_eq (vsub_ne_zero.2 hp₁) (vsub_ne_zero.2 hp₃)
+                                      (vsub_ne_zero.2 hp₄) (vsub_ne_zero.2 hp₆) hs
+
+/-- The unoriented angle at `p` between two points not equal to `p` is zero if and only if the
+unoriented angle is zero. -/
+lemma oangle_eq_zero_iff_angle_eq_zero {p p₁ p₂ : P} (hp₁ : p₁ ≠ p) (hp₂ : p₂ ≠ p) :
+  ∡ p₁ p p₂ = 0 ↔ ∠ p₁ p p₂ = 0 :=
+(o).oangle_eq_zero_iff_angle_eq_zero (vsub_ne_zero.2 hp₁) (vsub_ne_zero.2 hp₂)
+
+/-- The oriented angle between three points is `π` if and only if the unoriented angle is `π`. -/
+lemma oangle_eq_pi_iff_angle_eq_pi {p₁ p₂ p₃ : P} : ∡ p₁ p₂ p₃ = π ↔ ∠ p₁ p₂ p₃ = π :=
+(o).oangle_eq_pi_iff_angle_eq_pi
+
+/-- Swapping the first and second points in an oriented angle negates the sign of that angle. -/
+lemma oangle_swap₁₂_sign (p₁ p₂ p₃ : P) : (∡ p₂ p₁ p₃).sign = -(∡ p₁ p₂ p₃).sign :=
```

Comment:
I think you can swap the LHS and RHS and make this a simp lemma. Idem for the lemmas below.

Reply:
I've swapped the LHS and the RHS of these three lemmas in accordance with the general principle that the RHS should be the simpler side. I haven't made any of them `simp` lemmas, since there are three such lemmas with the same LHS and different RHS and no particular reason to prefer one as simpler than the others.